js/services.js

```diff
@@ -228,6 +229,21 @@ angular.module('wpApp.services', [])
 			});
 	}
 	
+	function getNextID() {
+		return $q.when( _db.allDocs( { descending: true, limit: 1 } ) )
+			.then( function( docs ) {
+				var nextID = 1;
+				if ( docs.total_rows == '1' ) {
+					var nextID = parseInt( docs.rows[0].id );
+					if ( nextID === '' ) {
```

Comment:
parseInt returns `NaN` if the value isn't a valid number. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

`nextID === ''` will always be `false`

A valid check would be `isNaN(nextID)`